src/prelude/string.savi

```diff
@@ -442,8 +442,7 @@
     )
     --res
 
-// TODO: Convert into `:struct`
-:class val StringPair
+:struct val StringPair
```

Comment:
I am looking forward to see a generic `Pair(T)` type.

Reply:
Added in https://github.com/savi-lang/savi/pull/122